mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -309,3 +309,11 @@ properties:
             full FHIR resource. When a resource change is too large or during heavy traffic, only the resource name will be
             sent. Clients should always check the "payloadType" label from a Pub/Sub message to determine whether
             it needs to fetch the full resource as a separate operation.
+        - !ruby/object:Api::Type::Boolean
```

Comment:
Can you add a test that specifies this in the update test for this resource? Just to check that the functionality works during update, especially going from specifying this field to not specifying it.

It can be added to the existing update test here: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/tests/resource_healthcare_fhir_store_test.go

Reply:
I think I added it correctly but let me know if there is something else you had in mind!